wordcloud/wordcloud.py

```diff
@@ -300,8 +318,15 @@ def process_text(self, text):
 
         return words
 
-    def generate(self, text):
-        """Generate wordcloud from text.
+    def generate(self, stuff):
```

Comment:
I would rather have different functions for different input formats, like ``generate_from_tokens`` or something like this. It is also not really clear from the docstring what input format is expected.